pkg/sentry/fsimpl/fuse/init.go

```diff
@@ -0,0 +1,299 @@
+// Copyright 2020 The gVisor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fuse
+
+import (
+	"math"
+
+	"gvisor.dev/gvisor/pkg/abi/linux"
+	"gvisor.dev/gvisor/pkg/sentry/kernel"
+	"gvisor.dev/gvisor/pkg/sentry/kernel/auth"
+)
+
+const (
+	// FUSE_MAX_TIME_GRAN_NS is the max value for the time granularity for file time stamps, 1s.
+	FUSE_MAX_TIME_GRAN_NS = 1000000000 
```

Comment:
I think only consts/flags in Linux header uses capital letter and underscores. Others in Go should use CamelCase. Also, if it is not used by other package, it should not be exported.

Reply:
Fixed. Thanks